app/views/candidate_mailer/application_submitted_apply_again.text.erb

```diff
@@ -1,15 +1,11 @@
 Dear <%= @application_form.first_name %>,
 
-# Application submitted
-
 You’ve submitted an application for <%= @application_choice.course_option.course.name_and_code %> at <%= @application_choice.course_option.course.provider.name %>.
 
 Your application reference is <%= @application_form.support_reference %>.
 
-You can track the progress of your application on your dashboard:
+Sign in to your account anytime to check the progress of your application:
```

Comment:
Argh, thought we'd already removed all references to 'dashboard'... 😱 

Not sure 'anytime' is needed? 

Should we also include a line to reassure candidates that we'll email them if there are any changes to their application? (To prevent them from thinking they need to constantly check)

```suggestion
Sign in to your account to check the progress of your application:
```